client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-status-content/lib/get-links.ts

```diff
@@ -63,10 +63,10 @@ const getLinks = (
 			break;
 		}
 		case 'plugin': {
-			link = `https://wordpress.com/plugins/updates/${ siteUrlWithMultiSiteSupport }`;
+			link = `https://wordpress.com/plugins/${ siteUrlWithMultiSiteSupport }`;
```

Comment:
This page requires more clicks to reach where they can update the plugins. Why not redirect them directly to the installed plugins page?

`${ siteUrlWithScheme }/wp-admin/plugins.php`

Reply:
Makes sense 👍